xapian-bindings/android/CROSS_COMPILE

```diff
@@ -55,14 +55,50 @@ make install
 
 The above configure will build xapian statically with position independent code.
 
+Generating the bindings for android
+===================================
+Now generate the bindings for android as follows:
+
+cd <path_to_xapian>/xapian-bindings/android
+mkdir -p org/xapian
+swig -Werror -I. -I../../xapian-core/include -c++ -java -module Xapian -package org.xapian -outdir org/xapian -o xapian_wrap.cc android.i
+
+This should generate xapian_wrap.cc,xapian_wrap.h and .java files in org/xapian. Now add the following line in org/xapian/XapianJNI.java at start of static block at line no 1114:
+
+System.loadLibrary("xapian_jni");
```

Comment:
Ooh, we can use that in the java bindings rather than modifying a SWIG-generated file with Perl.  Implemented that in 9f53a0a5761da1fb2f868c5da87cb374a6f5ce84.